golang/pkg/crane/k8s/secret.go

```diff
@@ -117,9 +115,34 @@ func (s *Secret) getSecretClient(namespace string) (v1.SecretInterface, error) {
 	return clientset.CoreV1().Secrets(namespace), nil
 }
 
-func (s *Secret) ApplyOpaqueSecret(ctx context.Context, namespace, name string, values map[string][]byte) (
+func (s *Secret) UpsertOpaqueSecrets(ctx context.Context, values map[string][]byte) (
```

Comment:
Apply is a proper kubectl instruction, to avoid confusion, I would rather use ApplyWithMerge or similar name.